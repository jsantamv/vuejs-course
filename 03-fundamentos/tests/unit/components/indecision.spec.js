import { shallowMount } from "@vue/test-utils"
import Indecision from '@/components/Indecision'

describe('Componont Indecision', () => {

    let wrapper

    beforeEach(() => {
        wrapper = shallowMount(Indecision)
    })

    test('debe hacer match con Snapshoot', () => {

        expect(wrapper.html()).toMatchSnapshot()
    })

})