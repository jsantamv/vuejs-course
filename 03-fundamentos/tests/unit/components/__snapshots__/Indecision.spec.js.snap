// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componont Indecision debe hacer match con Snapshoot 1`] = `
<!-- <img src="https://via.placeholder.com/250" alt="bg"> -->
<!--v-if-->
<div class="bg-dark"></div>
<div class="indecision-container"><input type="text" placeholder="Hazme una pregunta">
  <p>Recuerda terminar con signo de interrogacion (?)</p>
  <!--v-if-->
</div>
`;
